UnityProject/Assets/Scripts/Tilemaps/Behaviours/TilemapDamage.cs

```diff
@@ -76,7 +78,12 @@ private float DealDamageAt(float damage, AttackType attackType, Vector3Int cellP
 		}
 
 		data.AddTileDamage(Layer.LayerType, basicTile.Armor.GetDamage(damage < basicTile.damageDeflection? 0: damage, attackType));
-		SoundManager.PlayNetworkedAtPos(basicTile.SoundOnHit, worldPosition);
+
+		if (basicTile.SoundOnHit.IsReadyLoaded)
+		{
+			SoundManager.PlayNetworkedAtPos(basicTile.SoundOnHit, worldPosition);
+		}
```

Comment:
Bod, this probably isnt the best way, as im not sure if this blocks the sound from trying to load the first time, so when never to true. The issue was that it was spamming load but wasnt loading for some reason, it wasnt null or anything so is a bit weird.

Reply:
you could use a try catch to see if it's doing anything?

Reply:
Ive removed the check for now